index.html

```diff
@@ -201,68 +200,118 @@ <h2>Core definition</h2>
+                        </dd>
+                        <dt>Value space</dt>
+                        <dd>
+                            Similar to the Lexical space, <em>except</em> that the <code>dir</code> value is REQUIRED and can also take the value of <code>auto</code> (with the semantics as defined for the <a data-cite=!html/dom.html#the-dir-attribute><code>dir</code> attribute</a>   in&nbsp;[[html]]).
+                        </dd>
+                        <dt>Lexical to value mapping</dt>
+                        <dd>
+                            If the lexical value is <code>value@lang^dir</code> the mapping is the identity mapping. If <code>^dir</code> is missing, then the corresponding value space entry is <code>value@lang^auto</code>.
+                        </dd>
+                        <dt>The URL uniquely defining the new datatype</dt>
+                        <dd>
+                            The obvious URL would be to put this into the RDF namespace, i.e., <code>http://www.w3.org/1999/02/22-rdf-syntax-ns#LocalizableString</code>.
+                        </dd>
+                    </dl>
+                </section>
+                <section id="datatype-family">
+                    <h2>Define a <em>family</em> of language datatypes</h2>
+
+                    <p>A (very large, albeit finite) family of datatypes can be defined using the following URL patterns: <code>https://www.w3.org/i18n#XX{_YY}</code>, where:</p>
+
+                    <ul>
+                        <li>
+                            <strong>XX</strong>: is a [[bcp47]] language tag
+                        </li>
+                        <li>
+                            <strong>YY</strong>: is <code>ltr</code> or <code>rtl</code>, with the semantics as defined for the <a data-cite=!html/dom.html#the-dir-attribute><code>dir</code> attribute</a> in&nbsp;[[html]]. This part may be missing.</dd>                    
+                        </li>
+                     </ul>
+
+                    For each of these datatypes there is an identical lexical and value space:
+                    <dl>
+                        <dt>Lexical space</dt>
+                        <dd>
+                            Unicode [[UNICODE]] strings, which SHOULD be in Normal Form C&nbsp;[[nfc]].
+                        </dd>
+                        <dt>Value space</dt>
+                        <dd>
+                            Tuples consisting of the string, the `XX` tag extracted from the URL, and a directional tag, which is either the value of the `YY` or, if missing, the value of `auto`.
```

Comment:
I didn't push the idea that far, but that's clever. It would mean that, semantically speaking, each `i18n:XX_YY` datatype is a subtype of the future `rdf:langString`... So the new interpretation in RDF 1.2 could be argued to be a built-in inference, rather than a dirty hack... I like that :-)

Reply:
Yeah, that is always what I had in mind: all that `i18N:*` is just a kind of syntactic trick with some serialization-level microsyntax on the URL, but would not affect the core of RDF and its semantics...